plugin/xcodebuild.vim

```diff
@@ -3,13 +3,27 @@ command! XTest call <sid>test()
 
 let s:plugin_path = expand('<sfile>:p:h:h')
 
+function! s:bin_script(name)
+  return s:plugin_path . '/bin/' . a:name
+endfunction
+
+function! s:cli_args(...)
+  let cli_args = ''
+
+  for cli_arg in a:000
```

Comment:
Oh, wild. You actually need to pass the function argument as a string. vimscript, you crazy.